cmd/main.go

```diff
@@ -641,6 +642,17 @@ func main() {
 		}
 	}()
 
+	if Options.EnableKubeAPI {
+		// Splitting into API reader and writer interfaces as the reader bypasses the cache settings that are on the regular client
+		// and we need to be able to read secrets that we cannot match in cache settings
+		// The cachedApiClient can be used for writes as these are unaffected by cache.
+		apiReader := ctrlMgr.GetAPIReader()
+		cachedApiClient := ctrlMgr.GetClient()
+		localClusterImportOperations := localclusterimport.NewLocalClusterImportOperations(apiReader, cachedApiClient, log)
+		localClusterImport := localclusterimport.NewImportLocalCluster(&localClusterImportOperations, log, "local-cluster")
```

Comment:
if `local-cluster` is hard coded why do you need to add it a parameter?

Reply:
I guess we could make it a constant